architecture/adr-list/adr_002_identity_transactions.md

```diff
@@ -44,103 +44,191 @@ _**Note**: Hyperledger Indy also contains other transaction types beyond the one
 
-   ```text
-       {
-         "alias": "Alice DID",
-         "id": "GEzcdDLhCpGCYRHW82kjHd",
-         "verkey": "~HmUWn928bnFT6Ephf65YXv"
-       }
-   ```
+7. **`capabilityDelegation` (list of Verification Methods or strings with key aliases, optional):**
 
-#### **Update DID**
+8. **`service` (set of Service Endpoint maps, optional):**
 
-If there is no DID transaction with the specified DID \(`dest`\), it is considered as a creation request for a new DID.
+9. **`@context` (list of strings, optional):**
 
-If there is a DID transaction with the specified DID \(`dest`\), then this is update of existing DID. In this case, we can specify only the values we would like to override. All unspecified values remain the same. E.g., if a key rotation needs to be performed, the owner of the DID needs to send a DID transaction request with `dest`, `verkey` only. `alias` will stay the same.
+**Example:**
+```
+  {
+    "@context": [
+      "https://www.w3.org/ns/did/v1",
+      "https://w3id.org/security/suites/ed25519-2020/v1"
+    ],
+    "id": "N22KY2Dyvmuu2PyyqSFKue",
+    "authentication": [
+      {
+        "id": "N22KY2Dyvmuu2PyyqSFKue#z6MkecaLyHuYWkayBDLw5ihndj3T1m6zKTGqau3A51G7RBf3",
+        "type": "Ed25519VerificationKey2020", // external (property value)
+        "controller": "N22KY2Dyvmuu2PyyqSFKue",
+        "publicKeyMultibase": "zAKJP3f7BD6W4iWEQ9jwndVTCBq8ua2Utt8EEjJ6Vxsf"
+      }
+    ],
+    "capabilityInvocation": [
+      {
+        "id": "N22KY2Dyvmuu2PyyqSFKue#z6MkhdmzFu659ZJ4XKj31vtEDmjvsi5yDZG5L7Caz63oP39k",
+        "type": "Ed25519VerificationKey2020", // external (property value)
+        "controller": "N22KY2Dyvmuu2PyyqSFKue",
+        "publicKeyMultibase": "z4BWwfeqdp1obQptLLMvPNgBw48p7og1ie6Hf9p5nTpNN"
+      }
+    ],
+    "capabilityDelegation": [
+      {
+        "id": "N22KY2Dyvmuu2PyyqSFKue#z6Mkw94ByR26zMSkNdCUi6FNRsWnc2DFEeDXyBGJ5KTzSWyi",
+        "type": "Ed25519VerificationKey2020", // external (property value)
+        "controller": "N22KY2Dyvmuu2PyyqSFKue",
+        "publicKeyMultibase": "zHgo9PAmfeoxHG8Mn2XHXamxnnSwPpkyBHAMNF3VyXJCL"
+      }
+    ],
+    "assertionMethod": [
+      {
+        "id": "N22KY2Dyvmuu2PyyqSFKue#z6MkiukuAuQAE8ozxvmahnQGzApvtW7KT5XXKfojjwbdEomY",
+        "type": "Ed25519VerificationKey2020", // external (property value)
+        "controller": "N22KY2Dyvmuu2PyyqSFKue",
+        "publicKeyMultibase": "z5TVraf9itbKXrRvt2DSS95Gw4vqU3CHAdetoufdcKazA"
+      }
+    ]
+ }
+```
 
-**Note:** Fields `dest` and `owner` should have the same value.
 
-#### State format
+#### Verification Method
 
-`id -> {(alias, id, verkey), last_tx_hash, last_update_timestamp }`
+1. **`id` (string):**
 
-_DID transaction format:_
+   A string with format `<DIDDoc-id>#<key-alias`
 
-```text
-Did
-{
-    "alias": "Alice DID",
-    "dest": "GEzcdDLhCpGCYRHW82kjHd",
-    "verkey": "~HmUWn928bnFT6Ephf65YXv"
-}
+2. **`controller` (list of base58-encoded string):**
+
+   A set of identifier strings.
```

Comment:
```suggestion
   An identifier strings.
```

Reply:
It should be a single string. Sorry, my mistake, it needs to be fixed.